recipes/bw2speedups/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "bw2speedups" %}
+{% set version = "2.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "04cf55d9fdb74876fb690978966ca5901f2908a8d7607feb99b9e5fe361b8f72" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - bw2speedups
+
+about:
+  home: The package home page
+  license: BSD License
```

Comment:
It would be preferable if you could specify which flavor of the BSD license is using (e.g. `BSD 3-Clause`), but it doesn't look like it's specified upstream.